### PR TITLE
Create October 2023 Roundup

### PR DESCRIPTION
**Pull request by geekigirlx**
- Added 4 articles (last 4 under Articles & Learning) as we needed to bulk up. Can you let me know if any of them should be pulled/any descriptions are off? 
- Should this be listed under articles & learning or projects & tools? (Not sure if it’s a project) https://popovicu.com/posts/789-kb-linux-without-mmu-riscv/
- Same. asabove^ - is this in the right section under articles or should be a tool? - [**Linkable Loadable Extensions (LLEXT) — Zephyr Project Documentation**](https://docs.zephyrproject.org/latest/services/llext/index.html)<br>
- Is this ugly/better way to show this (first article)?
* [UBfuzz: Finding Bugs in Sanitizer Implementations](https://shao-hua-li.github.io/files/2024_ASPLOS_UBFUZZ.pdf) Detecting bugs in sanitizers with a fuzz- based approach. The caption below stuck with me. - Noah
GCC -O2 optimizes away the UB code, thus ASan cannot discover the UB.

### CODE REVIEW

File: _posts/2023-11-01-october-2023-roundup.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+
+
+## Upcoming Events
+
+- [**2023 IoT Online Conference**](https://www.iotonlineconference.com/)<br>
+The IoT Online Conference is back from November 14-16 and will feature talks from Memfault's Gillian Minnehan on GDB Deep Dive and Tyler Hoffman on Objectively Measuring the Reliability of IoT Devices. If you register now using our promo code, you will save 50% on registration fees! Use promo code: **MEMFAULTIOT** to drop your registration fee from $190 to $95. [Register here.](https://www.iotonlineconference.com/register.php)
+
+- [**Memfault Webinar - NXP + Memfault + Golioth: Bringing Observability and Device Management to IoT Devices**](https://hubs.la/Q026Gsz30)<br>
+Join us on Thursday, November 16th with [NXP](https://www.nxp.com/) and [Golioth](https://golioth.io/) for a joint webinar, as we dive into the future of IoT device management and observability. Can't make it? Register [here](https://hubs.la/Q026Gsz30) to get the record emailed to you afterward.
```
Comment (gminn) [inline _posts/2023-11-01-october-2023-roundup.md:73]: Small typo - record --> recording

**Review by gminn — COMMENTED** [review]
Looks good - just one comment and then my responses to your questions:
- https://popovicu.com/posts/789-kb-linux-without-mmu-riscv/ - articles
- [Linkable Loadable Extensions (LLEXT) — Zephyr Project Documentation](https://docs.zephyrproject.org/latest/services/llext/index.html) - tools
- I like the way you quoted the caption from Noah better than the plaintext one in the description! Stands out nicely

File: _posts/2023-11-01-october-2023-roundup.md (hunk 1 above)
Comment (geekigirlx) [reply _posts/2023-11-01-october-2023-roundup.md:73]: Thank you for the catch!

**Review by gminn — APPROVED** [review]
Looks great, thank you Matti! ❤️

**Pull request merged**